### PR TITLE
merge: DEV to QA (26/07/2024)

### PR DESCRIPTION
**Pull request by bhavanakarwade**
### What?
- Implemented prettyVc while bulk issuance

### Why?
- Passing prettyVc object while issuing credentials

### CODE REVIEW

File: apps/issuance/src/issuance.service.ts [hunk 1]
```diff
@@ -757,7 +762,8 @@ async sendEmailForCredentialOffer(sendEmailCredentialOffer: SendEmailCredentialO
             options: iterator.options
           }
         },
-        autoAcceptCredential: outOfBandCredential.autoAcceptCredential || 'always',
+        // For Educreds
```
Comment (GHkrishna) [inline apps/issuance/src/issuance.service.ts:765]: Do we need this comment?

File: apps/issuance/src/issuance.service.ts [hunk 2]
```diff
@@ -1501,15 +1540,19 @@ async sendEmailForCredentialOffer(sendEmailCredentialOffer: SendEmailCredentialO
...
-      const oobCredentials = await this.outOfBandCredentialOffer(
-        oobIssuancepayload, jobDetails?.platformName, jobDetails?.organizationLogoUrl
-      );
+      prettyVc = {
+        certificate: jobDetails?.certificate,
+        size: jobDetails?.size,
+        orientation: jobDetails?.orientation
+      };
 
+      oobIssuancepayload = await createOobJsonldIssuancePayload(JsonldCredentialDetails, prettyVc);
```
Comment (GHkrishna) [inline apps/issuance/src/issuance.service.ts:1552]:
```suggestion
      oobIssuancePayload = await createOobJsonldIssuancePayload(JsonldCredentialDetails, prettyVc);
```

**Review by GHkrishna — APPROVED** [review]
LGTM

**Review by pallavighule — APPROVED** [review]
LG

**Pull request merged**